Privacy.md

```diff
@@ -0,0 +1,56 @@
+# BlitzMail Privacy Policy
+This is the developer's policy which is created to give details and informs you if your personal information is collected, used, disclosed and secured by the developer. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. <hr>
+
+## Permission and Access
+
+BlitzMail software requires the storage permission to function effectively and efficiently. Without this permission enabled, you might be unable to perform certain operations with the software.
+BlitzMail has some other capabilities which are functions the software can remotely control on its own.
+
+- **have full network access**: The app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+
+- **view network connections**: This is the capability of the app to view information about network connections such as which networks exist and are connected.
```

Comment:
This is needed to send saved emails as soon as the user is connected to the internet again.